contrib/modelci/README.md

```diff
@@ -0,0 +1,47 @@
+# Model CI
+
+A simple Continuous Interation for Models, tracking the overall effect and performance.
+
+## Build testable model
+The new models should be located in `./models` directory, and has the following xonsh script files:
+
+- `train.xsh`
+- `validate.xsh`
+
+a model can run independently, and support following commands
+
+- `train.xsh --train_cost_out <path> --valid_cost_out <path>`, just train this model on CPU
+- `train.xsh --gpu <id> --train_cost_out <path>, --valid_cost_out <path>`, trian this model on a specific gpu.
+  - output the errors and time of the training of a model
+- `predict.xsh --out_path <path>`, load a model, and predict
+  - output the errors and time of the predicting of a model
+
+the log format should like this
+
+for `train_cost` and `valid_cost`, each line is
```

Comment:
Can we use VisualDL make a baseline, then every regression test just compare the several first mini batches results.